fastai/tabular/data.py

```diff
@@ -13,16 +13,16 @@ class TabularDataLoaders(DataLoaders):
     @classmethod
     @delegates(Tabular.dataloaders, but=["dl_type", "dl_kwargs"])
     def from_df(cls,
-                df:pd.DataFrame, # A Dataframe object
-                path:str='.', # Location of `df` (defaults to current working directory otherwise)
+                df:pd.DataFrame,
+                path:(str,Path)='.', # Location of `df` (defaults to current working directory otherwise)
                 procs:TabularProc=None, # List of `TabularProc`s (non-lazy `Transform`s) applied on object inplace
                 cat_names:list=None, # Column names pertaining to Categorical variables
                 cont_names:list=None, # Column names pertaining to Continuous variables
                 y_names:list=None, # Names of the Dependent variables
                 y_block:TransformBlock=None, # `TransformBlock` to use for the Target
                 valid_idx:list=None, # List of indices to use for the Validation set (defaults to a random split otherwise)
                 **kwargs):
-        "Create `TabularDataLoaders` from `df` in `path` using a list of `TabularProc`s"
+        "Create `TabularDataLoaders` from `df` in `path` using `procs`"
```

Comment:
reverted back to `procs` as i was repeating info as per suggestion from @muellerzr during live stream w/ w&b 😄

Reply:
I like that.  Good change